### DISCUSSION TITLE
sqoop import to s3 com.amazonaws.SdkClientException: Couldn't initialize a SAX driver to create an XMLReader

### DISCUSSION THREAD

**Discussion by Alxe1** [aws/aws-sdk-java#2800] [Q&A] [answered]
Using sqoop import mysql to s3, but it get `ERROR tool.ImportTool: Import failed: org.apache.hadoop.fs.s3a.AWSClientIOException: getFileStatus on tmp/tb_room_type3/: com.amazonaws.SdkClientException: Couldn't initialize a SAX driver to create an XMLReader: Couldn't initialize a SAX driver to create an XMLReader, ..., Caused by: org.xml.sax.SAXNotRecognizedException: Feature 'http://xml.org/sax/features/external-general-entities' is not recognized.`

the command is
```
sqoop import \
--connect jdbc:mysql://address/mydatabase \
--username xxx\
--password xxx\
--table tb_room_type \
--target-dir s3a://my-bucket/tmp/tb_room_type2/ \
--hive-table tb_room_type2 \
--hive-database mydatabase \
--fields-terminated-by "," \
--hive-import --m 1 \
--hive-overwrite 
```
How can I deal with it?

**Comment by debora-ito** [comment, marked as answer]
Hi @Alxe1 you'll have to reach out to the `sqoop` support team, as we don't know how they use the SDK internally.
